### ISSUE TITLE
Making PWM_RANGE / PWM_FREQ configurable

### ISSUE THREAD

**Issue by kianusch** [arendst/Sonoff-Tasmota#1025]
Hello,

Would be great, if one could configure PWM_FREQ and PWM_RANGE in the Tasmota Web interface.

Regards

**Comment by ionciubotaru** [comment]
Please use MQTT to configure your sonoff - works faster and you can use a script to configure all your devices

**Comment by KPK75** [comment]
PWMs are hardcoded and there are no commands to change them ( only recompile )
```
#define PWM_RANGE              1023         
#define PWM_FREQ               880
``` 
I would be "nice to have" them as options..

**Comment by arendst** [comment]
Will be in next (pre)release

**Comment by arendst** [comment]
Implemented in master release v5.9.0
